src/__tests__/read-test.js

```diff
@@ -18,6 +18,20 @@ describe("read", function() {
     ]);
   });
 
+  it("returns multiple records with utf8 chars", function() {
+    expect(read("6\n1234ß4\n12343\n123")).toEqual([
+      ["1234ß", "1234", "123"],
+      ""
+    ]);
+  });
+
+  it("returns multiple records with utf8 and emoji in many messages", function() {
+    expect(read("6\n1234ß5\nß23414\n🤷‍♂️.")).toEqual([
```

Comment:
I don't understand this test. It tells me the messages are:
6\n1234ß
5\nß2341
4\n🤷‍♂️.

1. The message two should be of  size 6 right? Like the first
2. The sequence of the latest message is pretty weird if I copy and paste.

Reply:
it is
6\n1234ß
5\nß234
14\n🤷‍♂️.

Reply:
14 for the last message. Ok.

Reply:
Does this also work when the emoji is the first message? I find this 14 unsettling.